Pipfile

```diff
@@ -13,8 +13,8 @@ pre-commit = "*"
 aws-sam-cli = "*"
 boto3 = "*"
 botocore = "*"
-requests = "*"
-crhelper = "*"
+requests = "~=2.23.0"
+crhelper = "~=2.0.6"
 
```

Comment:
+1 on adding version numbers -- don't want to do same for aws packages too?

Reply:
fixed